assets/queries/openAPI/general/path_parameter_not_required/query.rego

```diff
@@ -0,0 +1,29 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	version := openapi_lib.check_openapi(doc)
+	version != "undefined"
+
+	[path, value] := walk(doc)
+	param := value.parameters[n]
+	param.in == "path"
+	not_required(param)
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": sprintf("%s.parameters.name={{%s}}", [openapi_lib.concat_path(path), param.name]),
+		"issueType": "MissingAttribute",
```

Comment:
I think, if `param.required` is undefined, makes sense that attribute is missing, but when `param.required == false` should be categorized as `IncorrectValue`, no?

Reply:
Yes, it makes perfect sense 🙂